tests/python/frontend/pytorch/test_forward.py

```diff
@@ -3355,12 +3355,12 @@ def test_bincount():
     def test_fn(x, weights=None):
         return torch.bincount(x, weights=weights)
 
-    inp = torch.randint(0, 8, (5,), dtype=torch.int64)
-    weights = torch.linspace(0, 1, steps=5)
+    inp = torch.randint(0, 100, (10000,), dtype=torch.int64)
+    weights = torch.linspace(0, 100, steps=10000)
 
-    verify_trace_model(test_fn, [inp], ["llvm"])
-    verify_trace_model(test_fn, [inp, weights], ["llvm"])
-    verify_trace_model(test_fn, [inp, weights.to(torch.float64)], ["llvm"])
```

Comment:
I assume you removed this because the atomic add isn't precise enough for float64?

Reply:
No. For some reason, CUDA on CI fails to compile fp64 atomic add
https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-7044/3/pipeline/

I don't have this problem locally.